enterprise/reporting/noop.go

```diff
@@ -10,7 +10,7 @@ import (
 // NOOP reporting implementation that does nothing
 type NOOP struct{}
 
-func (n *NOOP) Report(metrics []*types.PUReportedMetric, txn *sql.Tx) {
+func (n *NOOP) Report(ctx context.Context, metrics []*types.PUReportedMetric, txn *sql.Tx) {
```

Comment:
```suggestion
func (*NOOP) Report(_ context.Context, _ []*types.PUReportedMetric, _ *sql.Tx) {
```

Possibly DeepSource might be complaining about the method receiver being there but not being used anyway.